intel_extension_for_transformers/neural_chat/pipeline/plugins/caching/cache.py

```diff
@@ -133,7 +133,9 @@ def init_similar_cache(self, data_dir: str = "api_cache", pre_func=get_prompt,
             config=config,
         )
 
-    def init_similar_cache_from_config(self, config_dir, embedding_model_dir):
+    def init_similar_cache_from_config(self, config_dir: str=os.path.join(
+            os.path.dirname(os.path.abspath(__file__)), "./cache_config.yaml"),
+            embedding_model_dir: str="hkunlp/instructor-large"):
```

Comment:
can we support other embedding model except instructor-large?

Reply:
current code does not support, `ValueError: Asking to pad but the tokenizer does not have a padding token. Please select a token to use as `pad_token` `(tokenizer.pad_token = tokenizer.eos_token e.g.)` or add a new pad token via `tokenizer.add_special_tokens({'pad_token': '[PAD]'})`.` I am improving this issue.

Reply:
we can support hkunlp/instructor-base, hkunlp/instructor-x1, and BERT-based models